src/test/java/edu/cornell/kfs/fp/batch/xml/AccountingXmlDocumentPojoTest.java

```diff
@@ -134,6 +155,16 @@ private void assertAccountingLineWasUnmarshalledCorrectly(
         assertEquals("Wrong line amount", expectedLine.getAmount(), actualLine.getAmount());
     }
 
+    private void assertItemWasUnmarshalledCorrectly(
+            AccountingXmlDocumentItem expectedItem, AccountingXmlDocumentItem actualItem) {
+        assertEquals("Wrong service date", expectedItem.getServiceDate(), actualItem.getServiceDate());

```

Comment:
I noticed you are doing this check in two different test classes/  Do you think it would be better to implement equals in AccountingXmlDocumentItem?

Reply:
One test class is validating the XML POJO items, the other class is validating the InternalBillingItem BOs. There may not be a practical way to create some common property-comparing code for both properties, unless the XML POJO's class hierarchy is changed or some reflection-based comparison is used instead. (InternalBillingItem doesn't appear to have an equals() method override of its own.)

Reply:
Sorry about the confusion, I see the difference now